debug/src/debug.js

```diff
@@ -74,6 +74,22 @@ export function initDebug() {
 
 		// Check prop-types if available
 		if (typeof vnode.type==='function' && vnode.type.propTypes) {
+			if (vnode.type.name === 'Lazy') {
```

Comment:
The string name might be unsafe because minifiers always mangle the function names. We should check for `displayName` here :+1: